tutorials/content/procedural_geometry.rst

```diff
@@ -14,7 +14,7 @@ This is the most common helper. :ref:`SurfaceTool<class_SurfaceTool>` is a class
 
 It has a similar API to OpenGL 1.x, and it's meant for static content. This means, the mesh is generated once and then used.
 
-Here is a simple example of how to use it to add a single triangle.
+Here is a simple example on how to use it to add a single triangle.
```

Comment:
"of" is the correct word to use here. It should not have been changed.

Reply:
i will fix this one too.